core/server/commons/enclave_data_directory/files_artifact_store.go

```diff
@@ -71,13 +64,16 @@ func NewFilesArtifactStoreForTesting(
 func (store FilesArtifactStore) StoreFile(reader io.Reader, contentMd5 []byte, artifactName string) (FilesArtifactUUID, error) {
 	store.mutex.Lock()
 	defer store.mutex.Unlock()
+	defer func(fileArtifactDb *file_artifacts_db.FileArtifactPersisted) {
+		_ = fileArtifactDb.Persist()
```

Comment:
I think that we should handle this error, because otherwise we could be missing data if something fails

Reply:
Done